lxd/backup.go

```diff
@@ -57,7 +58,7 @@ func backupCreate(s *state.State, args db.InstanceBackup, sourceInst instance.In
 		return tx.CreateInstanceBackup(ctx, args)
 	})
 	if err != nil {
-		if err == db.ErrAlreadyDefined {
+		if api.StatusErrorCheck(err, http.StatusConflict) {
```

Comment:
We can just return the err wrapped with something like `Failed creating backup record: %w`?

Reply:
Done.